deployed/goerli/addresses.json

```diff
@@ -0,0 +1,8 @@
+{
+  "BlockManager": "0x2E806452A18fEceBa9caF394A6558d17E157456b",
+  "JobManager": "0x4C0eb886933Eaa2ba22F1c464FC01151B578445e",
```

Comment:
This file is fine, but build/* files don't need to be commited as well. Those are generated files that are generated on every compile

Reply:
> but build/* files don't need to be committed as well. Those are generated files that are generated on every compile

We need ABIs to interact with deployed contracts. Before these ABIs were written to the `build` folder and being committed, but that was not appropriate since they can be changed during compilation.

We now keep the deployed ABIs and address per network, so they are unaffected by changes in the contract and the application can use them like before.

Reply:
ABI's have to be handled as part of the release process and tracked corresponding to the release version, network, etc.
Ideally they should be published via an npm package to be consumed by a frontend repo.

Reply:
We will keep committing build/* for now as a workaround till we migrate to a better solution.

Reply:
FYI we are in process of moving to a better solution here - https://github.com/razor-network/contracts/pull/68

Reply:
> We will keep committing build/* for now as a workaround till we migrate to a better solution.

Until we don't have a better solution, I think we should keep this one.
Here, `build` folder has been move to `deployed/goerli/abis` & `Address.md` is converted into `deployed/goerli/address.json` .

Its work well with the new migration script too.